src/components/marker-chart/MarkerChartEmptyReasons.js

```diff
@@ -56,7 +46,6 @@ const options: ExplicitConnectOptions<{||}, StateProps, {||}> = {
   mapStateToProps: (state: State) => ({
     thread: selectedThreadSelectors.getThread(state),
     threadName: selectedThreadSelectors.getFriendlyThreadName(state),
-    selectedTab: getSelectedTab(state),
   }),
   component: MarkerChartEmptyReasons,
 };
```

Comment:
note: @gregtatum changed this file a bit in https://github.com/devtools-html/perf.html/pull/1216 (it should land soon)